samples/nrf9160/serial_lte_modem/src/slm_at_gps.c

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <logging/log.h>
+LOG_MODULE_REGISTER(gps, CONFIG_SLM_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <net/socket.h>
```

Comment:
We use `nrf_sockets` here directly, I'd remove this inclusion to prevent confusion (unless it's really needed).

Reply:
Yes because of GPS. Removed net/socket